vagrant/roles/local.prep/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: copy vagrant generated inventory file - setup machine
   copy:
-    src: .vagrant/provisioners/ansible/inventory/vagrant_ansible_inventory
+    src: ./vagrant_ansible_inventory
```

Comment:
I had thought about using this chain of events to avoid duplication.
I was uncertain.
But I admit it looks better!